docs/src/main/asciidoc/datasource.adoc

```diff
@@ -101,7 +101,9 @@ For more information about pool size adjustment properties, see the <<jdbc-confi
 
 . Add the correct reactive extension for the database of your choice.
 
+ifndef::quarkus-reactive-db2-client[]
```

Comment:
does this not mean these are excluded by default?

Reply:
Hi, @maxandersen. Good question. `ifndef` displays the content in the community docs and suppresses it in the product docs, where the attribute will be defined.

Reply:
Some thoughts: We adopt a `ifndef` directive to assert attributes exclusively in the downstream environment:

- This method reveals content within the `ifndef` block to upstream viewers while concealing it from downstream readers.
- Conversely, it hides `ifdef` content from upstream and unveils it downstream.

This approach stands out for its simplicity and lower error margin than the alternative.

## The Alternative Strategy
The alternative—asserting attributes upstream and suppressing them downstream—introduces complexity and elevates the risk of significant issues:

- Erasing the attribute upstream by mistake can cause content to vanish from the community documentation. Such a deletion is a more common error than an unintended attribute assertion upstream.
- It's more challenging to ensure the reliable suppression of an upstream attribute once it's downstream compared to directly asserting an attribute downstream. This increases the likelihood of operational failures.

Reply:
+1, i'd say we prefix "no-" to such conditionals.

Reply:
> +1, i'd say we prefix "no-" to such conditionals.
Updated. Thanks.